app/commands_app/commands/status.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import os
+
+from redis.exceptions import RedisError
+from redis_util.redis_connection import ping_redis
+
+def parse_args(sub_parser):
+    subparser = sub_parser.add_parser("status", help="Report on the status of the application")
+    # register the function to be executed when command "status" is called
+    subparser.set_defaults(func=print_status)
+
+def print_status(arg):
+    """ Prints the status of the aladdin-demo pod and the redis pod """
+    print_aladdin_demo_status()
+    print_redis_status()
+
+def print_aladdin_demo_status():
+    print("pinging aladdin-demo ...")
+    host = os.environ["ALADDIN_DEMO_SERVICE_HOST"]
+    port = os.environ["ALADDIN_DEMO_SERVICE_PORT"]
+    url = "http://{}:{}/ping".format(host, port)
+    try: 
+        r = requests.get(url)
+        if r.status_code == 200:
+            print("aladdin demo endpoint ready")
```

Comment:
maybe i'm wrong, but I don't think ready is the right word here - maybe we should print "aladdin demo ping successful" or something like that